addons/mass_mailing/i18n/mass_mailing.pot

```diff
@@ -4408,3 +4408,42 @@ msgstr ""
 #: model_terms:ir.ui.view,arch_db:mass_mailing.s_rating_options
 msgid "⌙ Inactive"
 msgstr ""
+
+#. module: mass_mailing
+#: model_terms:ir.ui.view,arch_db:mass_mailing.res_config_settings_view_form
+msgid "Check how well your mailing is doing a day after it has been sent"
```

Comment:
Seems it was done manually ^^ as alphabetical ordering is not respected. Either you do it manually with alphabetical ordering, either you do it through the "export translation" menu. Using ``git add -p`` you can keep only the new strings and strip out noisy changes.

Reply:
Hi! Hopefully I did it right this time! (First time working with translations...)